extensions/wikia/WikiFactory/Hubs/WikiFactoryHub.php

```diff
@@ -175,6 +175,10 @@ public function getCategoryId( $city_id ) {
 	 */
 
 	public function getVerticalId( $city_id ) {
+		global $wgWikiaEnvironment;
+		if ( $wgWikiaEnvironment == WIKIA_ENV_INTERNAL ) {
+			$city_id = 11;
```

Comment:
Created ticket for cleanup ```11```s in our codebase: https://wikia-inc.atlassian.net/browse/MAIN-3586